BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -401,6 +431,27 @@ public final class BlueprintView: UIView {
         return rootController.children
     }
 
+    /// Forces a synchronous layout, for testing purposes.
+    @_spi(BlueprintDebugging)
+    public func forceSynchronousLayout() {
+        setNeedsViewHierarchyUpdate()
+        updateViewHierarchyIfNeeded()
+    }
+
+    /// Dumps the result of the most recent layout, by recursing through the layout tree and calling
+    /// the provided visitor on each node. By default, this prints to `stdout`.
+    @_spi(BlueprintDebugging)
+    public func dumpLayoutResult(
```

Comment:
I think this is somewhat duplicated with the impl on `LayoutResultNode`, can we just call into that impl?

Reply:
> I think this is somewhat duplicated with the impl on `LayoutResultNode`, can we just call into that impl?

Sure, I think at one point it was handy to have both, but we don't need them both now.

Reply:
I'm going to defer small things like this to a follow-up PR to avoid having to rebase the rest of the stack.